pkg/kubelet/networks.go

```diff
@@ -54,6 +55,8 @@ func (nh *networkHost) SupportsLegacyFeatures() bool {
 // methods, because networkHost is slated for deletion.
 type criNetworkHost struct {
 	*networkHost
+	// criNetworkHost currently support legacy features. Hence no need to support PortMappingGetter
```

Comment:
Why do we still need the `networkHost` (i.e., a back channel to kubelet) now that host port support has been added to CRI? https://github.com/kubernetes/kubernetes/blob/v1.6.0-alpha.1/pkg/kubelet/networks.go#L27